Firestore/Source/Local/FSTLevelDB.mm

```diff
@@ -33,9 +34,11 @@
 NS_ASSUME_NONNULL_BEGIN
 
 static NSString *const kReservedPathComponent = @"firestore";
+static FSTLevelDBSchemaVersion kSchemaVersion = 1;
```

Comment:
Given that the migrations are defined in FSTLevelDBMigrations should this constant live there?

That way you can add comments to that implementation that indicate that that relationship should be to the numbers in the switch block (similar to the guidance given in the Android SQLiteSchema)

Reply:
Done.